pkgs/applications/audio/AMB-plugins/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     longDescription = ''
       Mono and stereo to B-format panning, horizontal rotator, square, hexagon and cube decoders.
     '';
-    version = "${version}";
+    version = version;
```

Comment:
While we’re here, could change these to `inherit`?

Reply:
I would make it as a separate rule, there could be more `version = version;` case